### PR TITLE
Base L2 deployment

### PR DESCRIPTION
**Pull request by fenris85**
Base L2 deployment: 

- ServiceRegistry
- DMA core contracts
- DMA Swap
- DPM
- DSProxy

### CODE REVIEW

File: packages/deploy-configurations/configs/base.conf.ts [hunk 1]
```diff
@@ -0,0 +1,1211 @@
...
+      },
+      DSGuardFactory: {
+        name: 'DSGuardFactory',
+        deploy: true,
+        address: '0x89010CBE61Ba3A800aFDc20D22E8163c6256E5Ef',
+        serviceRegistryName: SERVICE_REGISTRY_NAMES.common.DS_GUARD_FACTORY,
+        history: [],
+        constructorArgs: []
+      },
+      AccountGuard: {
```
Comment (paszkowskiDamian) [inline packages/deploy-configurations/configs/base.conf.ts]: Do we need those maker contracts?

File: packages/deploy-configurations/configs/base.conf.ts [hunk 2]
```diff
@@ -0,0 +1,1211 @@
...
+    },
+    WLD: {
+      name: 'WLD',
+      address: '0x0000000000000000000000000000000000000000'
+    },
+    WSTETH: {
+      name: 'WSTETH',
+      address: '0x0000000000000000000000000000000000000000',
+      serviceRegistryName: SERVICE_REGISTRY_NAMES.common.WSTETH
+    },
```
Comment (paszkowskiDamian) [inline packages/deploy-configurations/configs/base.conf.ts]: Can we override later those addresses in registry?

File: packages/deploy-configurations/configs/base.conf.ts (hunk 1 above)
Comment (fenris85) [reply packages/deploy-configurations/configs/base.conf.ts]: Probably we don't need it. But it's deployed already. We could remove it from config for the future (in all networks or on L2s only)

File: packages/deploy-configurations/configs/base.conf.ts (hunk 2 above)
Comment (fenris85) [reply packages/deploy-configurations/configs/base.conf.ts]: We cannot override in current ServiceRegistry implementation. But this one (or any empty) is not added to SR.

**Review by halaprix — APPROVED** [review]

**Pull request merged**